lib/open_food_network/packing_report.rb

```diff
@@ -146,8 +147,8 @@ def is_by_customer?
       params[:report_type] == "pack_by_customer"
     end
 
-    def customer_code(email)
-      customer = Customer.where(email: email).first
+    def customer_code(order)
+      customer = order.customer
```

Comment:
This is fixing a bug as well. The report could have fetched the customer code given by another enterprise.